<launch>

  <node pkg="robot_control_base" type="follow_line.py" name="follow_line" output="screen">
    <param name="reference_frame" value="/world"/>
    <param name="body_frame" value="/rosControlledBubbleRob"/>
    <param name="dist_threshold" value="0.1"/>
    <param name="velocity" value="0.5"/>
    <param name="k_y" value="4.0"/>
    <param name="k_theta" value="2.0"/>
    <param name="max_angular_speed" value="1.5"/>
    <param name="max_y_error" value="2.0"/>
  </node>

</launch>
